Creative/apps/creative-tv/package.json

```diff
@@ -44,6 +44,7 @@
     "graphql-request": "^5.1.0",
     "graphql-tag": "^2.12.6",
     "iron-session": "^6.3.1",
+    "mongodb": "^6.3.0",
```

Comment:
So we're going mongoDB huh 🤔 Curious why did you choose to take this route?

Reply:
At the moment; it would help persist the contract address as per `ERC1155` standard we spoke about. It is temporal till we come up with better option... I though ipfs.. but .... till then